CallAutomation_OutboundCalling/CallAutomation_OutboundCalling/Program.cs

```diff
@@ -45,55 +72,68 @@
     foreach (var cloudEvent in cloudEvents)
     {
         CallAutomationEventBase parsedEvent = CallAutomationEventParser.Parse(cloudEvent);
-        logger.LogInformation($"{parsedEvent?.GetType().Name} parsedEvent received for call connection id: {parsedEvent?.CallConnectionId}");
+        logger.LogInformation(
+                    "Received call event: {type}, callConnectionID: {connId}, serverCallId: {serverId}",
+                    parsedEvent.GetType(),
+                    parsedEvent.CallConnectionId,
+                    parsedEvent.ServerCallId);
+
         var callConnection = callAutomationClient.GetCallConnection(parsedEvent.CallConnectionId);
         var callMedia = callConnection.GetCallMedia();
 
-        if (parsedEvent is CallConnected)
+        if (parsedEvent is CallConnected callConnected)
         {
             logger.LogInformation($"Start Recording...");
             CallLocator callLocator = new ServerCallLocator(parsedEvent.ServerCallId);
             var recordingResult = await callAutomationClient.GetCallRecording().StartAsync(new StartRecordingOptions(callLocator));
             recordingId = recordingResult.Value.RecordingId;
 
+            var choices = GetChoices();
```

Comment:
Lets add log statements here.. like Recording Started, and then Fetching RecognizeOptions etc